WelcomePage/WelcomePage.js

```diff
@@ -0,0 +1,15 @@
+const inpName = document.getElementsByClassName('inpName')[0];
+const btnLogin = document.getElementsByClassName('btnLogin')[0];
+
+btnLogin.onclik = welcome;
+
+function welcome(e){
+    if(inpName.value == '') 
+        alert('Enter Your Name Please');
```

Comment:
I think internal {} needed here?

Reply:
when if condition take one statement, we don't need to put {}

Reply:
New Info, thank You